src/components/ProductDetail.js

```diff
@@ -23,11 +23,8 @@ function ProductDetail(props) {
             <div className="ratings">
                 <p className="pull-right">{props.product.reviews}</p>
                 <p>
-                    <span className="glyphicon glyphicon-star"></span>
-                    <span className="glyphicon glyphicon-star"></span>
-                    <span className="glyphicon glyphicon-star"></span>
-                    <span className="glyphicon glyphicon-star"></span>
-                    <span className="glyphicon glyphicon-star"></span>
+                    {starRatings(props.product.rating)}
```

Comment:
You are close on this, but when you use a function inline with jsx, that function needs to return some jsx
your function starRatings is not returning anything so calling it inline with jsx is not accomplishing anything